src/modules/system/users/components/UserEditingModal.vue

```diff
@@ -173,7 +173,17 @@ const handleCreateUser = async () => {
               label="用户名"
               type="text"
               name="name"
-              validation="required|alphanumeric|length:0,50"
+              :validation="[
+                ['required'],
+                ['length:0,50'],
+                [
+                  'matches',
+                  /^[a-z0-9]([-a-z0-9]*[a-z0-9])?(.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/,
```

Comment:
`.` 需要转义
```suggestion
                  /^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/,
```
 测试效果如下
<img width="727" alt="image" src="https://user-images.githubusercontent.com/38999863/220233066-4cb9d2c9-a40e-43e3-84f1-f92009b75854.png">